index.php

```diff
@@ -126,7 +126,7 @@
 	$mybb->settings['uploadspath'] = MYBB_ROOT.$mybb->settings['uploadspath'];
 }
 
-require_once MYBB_ROOT."inc/class_xml.php";
+require_once MYBB_ROOT."inc/class_xmlparser.php";
```

Comment:
I guess we may discard this line and use [`create_xml_parser()`](https://github.com/mybb/mybb/blob/mybb_1829/inc/functions.php#L9417), already provided by MyBB a few versions ago, whenever we want a XML parser.